SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,27 +1,10 @@
 #!/bin/bash
 
-# Create a middle layer for installing ipykernel, putting it apart from the user environment
```

Comment:
The commit message still needs to reflect the fact that venv-pack does not support editable packages, and the middle layer was based on the use of a .pth file, which is seen as a source of editable packages and therefore makes venv-pack fail.